giraffe/src/types/index.ts

```diff
@@ -916,3 +924,44 @@ export interface AnnotationMark {
   pin: AnnotationPinType
   id?: string
 }
+
+export enum ComponentColor {
+  Default = 'default',
+  Primary = 'primary',
+  Secondary = 'secondary',
+  Tertiary = 'tertiary',
+  Success = 'success',
+  Warning = 'warning',
+  Danger = 'danger',
+}
+
+export enum Alignment {
+  Left = 'left',
+  Center = 'center',
+  Right = 'right',
+}
+
+export enum VerticalAlignment {
+  Top = 'top',
+  Middle = 'middle',
+  Bottom = 'bottom',
+}
+
+export enum ButtonShape {
+  Default = 'none',
+  Square = 'square',
+  StretchToFit = 'stretch',
+}
+
+export enum ButtonType {
+  Button = 'button',
+  Submit = 'submit',
+}
+
+export enum BorderType {
+  None = 'none',
+  Horizontal = 'horizontal',
+  Vertical = 'vertical',
+  Both = 'both',
+  All = 'all',
+}
```

Comment:
If these are used only by SimpleTable then they should be moved to `giraffe/src/components/SimpleTable/Table/index.tsx` or if needed elsewhere in the `SimpleTable` hierarchy then one folder up is fine too. These don't belong in the global index file though.